grails-app/conf/Config.groovy

```diff
@@ -407,9 +407,6 @@ jbrowse {
 //	// Warning: We are still testing the performance of the NeatFeatures plugins in combination with Apollo.
 //	// We advise caution if enabling these plugins with Apollo until this process is finalized.
     plugins {
-        WebApollo {
```

Comment:
I haven't had a chance to test, yet.  Is this inferred because we are already in Apollo?